### PR TITLE
coefficients export: set default coefficient to 1 if passport is not used

### PR DESCRIPTION
**Pull request by gravityblast**

### CODE REVIEW

File: src/http/api/v1/exports.ts [hunk 1]
```diff
@@ -109,14 +109,23 @@ async function exportVoteCoefficientsCSV(db: JsonStorage, round: Round) {
     }),
   ]);
 
+  const usePassport =
```
Comment (boudra) [inline src/http/api/v1/exports.ts:112]: is it a react hook?

**Review by boudra — APPROVED** [review]

File: src/http/api/v1/exports.ts (hunk 1 above)
Comment (gravityblast) [reply src/http/api/v1/exports.ts:112]: @boudra  yeah it's defined server side but it runs on a remote browser.

Comment (hmrtn) [reply src/http/api/v1/exports.ts:112]: Mo is right! This should be conventionally boolean name like - such as `isPassport` or `isSybilDefended`, etc.

**Comment by boudra** [comment]
whoops build is failing

**Comment by gravityblast** [comment]
@boudra I fixed a eslint error, it's rebuilding

File: src/http/api/v1/exports.ts (hunk 1 above)
Comment (gravityblast) [reply src/http/api/v1/exports.ts:112]: @hmrtn @boudra  this var tells the rest of the program if we need to use passport or not. 
`isPassport` wouldn't make sense.

would you prefer `isPassportEnable`?

Comment (boudra) [reply src/http/api/v1/exports.ts:112]: `isPassportEnabled` sounds good

Comment (gravityblast) [reply src/http/api/v1/exports.ts]: I updated it to `isPassportEnable`

**Review by hmrtn — DISMISSED** [review]

**Review by boudra — APPROVED** [review]

**Pull request merged**